File: backend/src/main/java/io/chesslave/hands/MoverException.java
package io.chesslave.hands;

public class MoverException extends RuntimeException {
    private static final long serialVersionUID = 2338005287898395747L;

    public MoverException(Throwable cause) {
        super(cause);
    }

    public MoverException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: backend/src/main/java/io/chesslave/hands/MoverException.kt
package io.chesslave.hands

class MoverException : RuntimeException {

    constructor(cause: Throwable) : super(cause) {
    }

    constructor(message: String, cause: Throwable) : super(message, cause) {
    }

    companion object {
        private val serialVersionUID = 2338005287898395747L
    }
}
